<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>       退出登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/hotel1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/hotel1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/hotel1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>          注册酒店</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;
</string>
    </property>
    <property name="text">
     <string>您已拥有的酒店 : )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/people1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/people1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/people1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>       添加雇员</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="HotelPass">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文楷体&quot;;
background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>经营许可证</string>
     </property>
     <property name="background">
      <color alpha="100">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店名字</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店特色</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店星级</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>拥有者</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/room1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/room1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/room1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>         查看房间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;
</string>
    </property>
    <property name="text">
     <string>待审核的酒店: (</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="HotelUnPass">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文楷体&quot;;
background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>经营许可证</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color alpha="0">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店名字</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color alpha="0">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店地址</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店特色</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>酒店星级</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>拥有者</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>状态</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="newManager">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 12pt &quot;Calibri&quot;;
border:2px groove gray;border-radius:10px;padding:2px 4px</string>
    </property>
   </widget>
   <widget class="QListWidget" name="seeManager">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Calibri&quot;;
border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color:rgba(0,0,0,0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;
</string>
    </property>
    <property name="text">
     <string> 您已雇佣的经理: )</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>:/myimages/view.jpg.png</normaloff>:/myimages/view.jpg.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>关闭(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
